website/assets/js/pages/docs/basic-documentation.page.js

```diff
@@ -82,16 +82,18 @@ parasails.registerPage('basic-documentation', {
 
   mounted: async function() {
     // Algolia DocSearch
-    docsearch({
-      appId: 'NZXAYZXDGH',
-      apiKey: 'f3c02b646222734376a5e94408d6fead',
-      indexName: 'fleetdm',
-      inputSelector: (this.isDocsLandingPage ? '#docsearch-query-landing' : '#docsearch-query'),
-      debug: false,
-      algoliaOptions: {
-        'facetFilters': ['section:docs']
-      },
-    });
+    if(this.docSearchPublicKey) { // Note: Docsearch will only be enabled if sails.config.custom.docSearchPublicKey is set.
```

Comment:
What happens if it is not enabled?  (My guess is that the search will just be there but not work, right?  Worth making that explicit in the comment, so we understand what, if any, impact it has on the CSS layout during development if not configured.  And so our future selves or other contributors don't come in here and make the wrong assumption when troubleshooting)